manifests/a/AliceTeam/CarnegieMellonUniversity/Alice3/3.7.0.0.876/AliceTeam.CarnegieMellonUniversity.Alice3.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AliceTeam.CarnegieMellonUniversity.Alice3
+PackageVersion: 3.7.0.0.876
+PackageLocale: en-US
+Publisher: Alice Team - Carnegie Mellon University

```

```suggestion
Publisher: Alice Team, Carnegie Mellon University
```
Must match Control Panel/Apps&Features